src/main/java/org/openstreetmap/atlas/checks/commands/AtlasChecksLogDiffSubCommand.java

```diff
@@ -78,19 +82,28 @@ public AtlasChecksLogDiffSubCommand()
     }
 
     /**
-     * Get the atlas ids from an array of features.
+     * Get the unique ids for a flag. Fall back to getting the atlas ids from the features for
+     * reverse compatibility.
      *
-     * @param features
-     *            a {@link JsonArray} of features
+     * @param flagJson
+     *            a {@link JsonObject} of a flag
      * @return a {@link Set} of {@link String} ids
      */
-    private Set<String> getAtlasIdentifiers(final JsonArray features)
+    private Set<String> getIdentifiers(final JsonObject flagJson)
     {
-        return Iterables.stream(features)
-                .filter(object -> object.getAsJsonObject().get(PROPERTIES).getAsJsonObject()
-                        .has(IDENTIFIER))
-                .map(object -> object.getAsJsonObject().get(PROPERTIES).getAsJsonObject()
-                        .get(IDENTIFIER).getAsString())
-                .collectToSet();
+        final JsonObject flagProperties = flagJson.get(PROPERTIES).getAsJsonObject();
+        return flagProperties.has(IDENTIFIERS)
+                ? Iterables.stream(flagProperties.get(IDENTIFIERS).getAsJsonArray())
+                        .map(JsonElement::getAsString).collectToSet()
+                : Iterables.stream(flagJson.get(FEATURES).getAsJsonArray()).filter(object -> object
+                        .getAsJsonObject().get(PROPERTIES).getAsJsonObject().has(IDENTIFIER)
+                        || object.getAsJsonObject().get(PROPERTIES).getAsJsonObject().has("ItemId"))
```

Comment:
This is a bit of a funny one in that it is not a standard geojson key. It should probably be defined as a public constant somewhere in the output event classes. There are a bunch of similar items in the event classes that could use a refactor. It would probably be best to have a separate PR to update all of them.

Reply:
In looking at this again I have realized this is the old identifier key that CheckFlagEvent used to use. It is still in this tool for reverse compatibility.